crates/red_knot_python_semantic/mdtest.py

```diff
@@ -158,15 +163,18 @@ def watch(self) -> Never:
         self._run_mdtest()
         self.console.print("[dim]Ready to watch for changes...[/dim]")
 
-        for changes in watch(CRATE_ROOT):
+        for changes in watch(*DIRS_TO_WATCH):
             new_md_files = set()
             changed_md_files = set()
             rust_code_has_changed = False
 
             for change, path_str in changes:
                 path = Path(path_str)
 
-                if path.suffix == ".rs":
+                # Obviously a change to a vendored stub file isn't really "Rust code",
+                # but we need to rebuild the binary to pick up the changes,
+                # # so it comes to the same thing
```

Comment:
This is true, but I still think that to avoid confusing future-us unnecessarily, it warrants a change to the variable name `rust_code_has_changed` (`rust_code_or_typeshed_has_changed`), and the user-facing message "Rust code has changed, recompiling tests..." below ("Rust code or vendored typeshed has changed, recompiling tests...".